src/features/unified-share-modal/flowTypes.js

```diff
@@ -171,6 +171,7 @@ export type sharedLinkType = {
 
 export type collaboratorType = {
     collabID: number,
+    email: string,
```

Comment:
Is email always present, or should the type allow it to be undefined? I see in some of the tests that groups don't have an email field